media/stylus/components/compat-tables/bc-legend.styl

```diff
@@ -0,0 +1,26 @@
+/*
+legend
+====================================================================== */
+
+.bc-legend {
+    vendorize(column-count, 2);
+    page-break-inside: avoid;
+    webkit-column-break-inside: avoid;
+    break-inside: avoid;
```

Comment:
CSS3 Multiple column layout not supported by: Firefox (37,38), Chrome (42,43), Safari (8,7.1), Opera (29,30), iOS Safari (8,8.1-8.3)